pennylane/transforms/mitigation.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the mitigation transform
+"""
+# pylint: disable=import-outside-toplevel,protected-access
+import pennylane as qml
+from pennylane.operation import Operation
+
+cirq_operation_map = None
+
+
+def _load_operation_map():
+    """If not done already, this function loads the mapping between PennyLane operations and Cirq
+    operations.
+    """
+    global cirq_operation_map
+    if cirq_operation_map is None:
+        from pennylane_cirq.cirq_device import CirqOperation, CirqDevice
+
+        operation_map = CirqDevice._operation_map
+        inv_operation_map = {}
+
+        for key in operation_map:
+            if not operation_map[key]:
+                continue
+
+            inverted_operation = CirqOperation(operation_map[key].parametrization)
+            inverted_operation.inv()
+
+            inv_operation_map[key + Operation.string_for_inverse] = inverted_operation
+
+        cirq_operation_map = {**operation_map, **inv_operation_map}
+
+    return cirq_operation_map
+
+
+def _tape_to_cirq(tape):
```

Comment:
I think this is a useful enough transform that it deserves not only to be public, but to be in its own module

Reply:
The cool thing is that we have a lot of these transformations already, just wrapped up in a device (e.g., this was from PennyLane-Cirq)

Reply:
yes, was thinking the same thing!